### DISCUSSION TITLE
Cannot find module '@intlify/vite-plugin-vue-i18n/messages' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by perkrlsn** [intlify/bundle-tools#66] [Q&A]
When trying to import the `messages` default export from the`@intlify/vite-plugin-vue-i18n/messages` I keep getting this type error.

I've tried adding `@intlify/vite-plugin-vue-i18n` to the `types` array in the `tsconfig.json`, but I'm stumped. My only solutions so far is to copy the declaration defined in `client.d.ts` to a local `d.ts` file in my project.

![image](https://user-images.githubusercontent.com/12989781/139639407-199e753d-bb77-452b-abfb-0d5697562bcf.png)

**Comment by kazupon** [comment]
You need to configure tsconfig
you can see the docs:
https://github.com/intlify/bundle-tools/tree/main/packages/vite-plugin-vue-i18n#client-types
